pymc4/distributions/__init__.py

```diff
@@ -1 +1,8 @@
-from . import base
+__backend = 'tensorflow'
+
+if __backend == 'base':
```

Comment:
Base backend is not runnable, I see no case when we might need to load it

Reply:
removed.